packages/cozy-harvest-lib/package.json

```diff
@@ -31,6 +31,7 @@
     "cozy-bi-auth": "0.0.19",
     "cozy-doctypes": "^1.79.0",
     "cozy-logger": "^1.7.0",
+    "cozy-realtime": "3.12.2",
```

Comment:
For now, realtime was a peerDep of harvest. Do we want to add it as a dep?

Reply:
Nope, removed, kept as peer dependency, thanks !